pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, gtk2 , pkgconfig , python27 , gfortran , python27Packages , lesstif , cfitsio , getopt , perl , groff }:
+
+stdenv.mkDerivation rec {
+  version = "feb17d";
```

Comment:
OK, already suggested to my contact in the Gildas team. I think that I'll do something simple for now and see if they changed the thing for the next release